Makefile.am

```diff
@@ -206,9 +206,6 @@ bots:
 multi-word: po/cockpit.pot
```

Comment:
This one too, I think?

Reply:
I am not going to pretend that I left it on purpose - but I found it useful now that I could easily grab all multi world strings and then check them for weirdness. But I can drop it indeed.

Reply:
Dropped it and fixed few strings